src/Microsoft.AspNetCore.Mvc.Core/RequireHttpsAttribute.cs

```diff
@@ -15,15 +15,26 @@ namespace Microsoft.AspNetCore.Mvc
     [AttributeUsage(AttributeTargets.Class | AttributeTargets.Method, Inherited = true, AllowMultiple = false)]
     public class RequireHttpsAttribute : Attribute, IAuthorizationFilter, IOrderedFilter
     {
+        private bool? _permanent = null;
+
         /// <summary>
         /// Specifies whether a permanent redirect, <c>301 Moved Permanently</c>,
         /// should be used instead of a temporary redirect, <c>302 Found</c>.
         /// </summary>
-        public bool Permanent { get; set; }
+        public bool Permanent {
+            get
+            {
+                return _permanent ?? false;
```

Comment:
when i use it  to mark a controller or action an error occurs when specify a value ,  is not a valid attribute parameter type because is nullable value.